notebooks/01-introduction-to-elements.ipynb

```diff
@@ -0,0 +1,584 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href='http://www.holoviews.org'><img src=\"assets/header_logo.png\" alt=\"HoloViews logo\" width=\"20%;\" align=\"left\"/></a>\n",
+    "<div style=\"float:right;\"><h2>01. Introduction to Elements</h2></div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preliminaries\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the  ``hvtutorial`` environment has been correctly created and activated using the instructions listed on the [index](00-index.ipynb) page, the following imports should run and ``hv.extension('bokeh')``  should present a small HoloViews logo:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import holoviews as hv\n",
+    "hv.extension('bokeh')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we import numpy and pandas with their standard aliases and HoloViews with its standard short form of ``hv``. The line reading ``hv.extension('bokeh')`` loads and activates the bokeh plotting backend so all visualizations will be generated using [Bokeh](bokeh.pydata.org). We will see how to use matplotlib instead of bokeh later in the tutorial [XXX]."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# What are elements?\n",
+    "\n",
+    "In short, elements are HoloViews' most basic, core primitives that are all subclasses of ``Element``. All ``Element`` objects accept semantic metadata that allows their input data to be given an automatic, visual representation. Most importantly, element objects always preserve the raw data they are supplied.\n",
+    "\n",
+    "In this notebook we will explore a number of different element types and examine some of the ways that elements can supplement the supplied data with useful semantic data. There are many element types you might chose in the exercises and to help you make your choice, you can browse the [visual reference](http://holoviews.org/reference/index.html) which shows all the element types."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating elements\n",
+    "\n",
+    "All basic elements accept their data as a single, mandatory positional argument which may be supplied in a number of different formats, some of which we will now examine. A handful of *annotation* elements are exceptions to this rule, namely ``Arrow``, ``Text``, ``Bounds``, ``Box`` and ``Ellipse`` as they require additional positional arguments.\n",
```

Comment:
Probably a simpler way to say this is something like:

    All elements (apart from Annotations) accept a single, mandatory positional 
    argument -- "the data".  Various different types and formats of data are 
    accepted, as we will start to explore below.